account_invoice_change_currency/tests/test_account_invoice_change_currency.py

```diff
@@ -146,17 +155,17 @@ def test_change_invoice_currency(self):
         expected_value = before_curr._convert(
             before_amount, after_curr, inv.company_id, fields.Date.today()
         )
-
         self.assertEqual(
-            float_compare(inv.amount_total, expected_value, 1),
+            float_compare(inv.amount_total, expected_value, 0),
             0,
             "Total amount of invoice does not equal to expected value!!!",
         )
 
     def test_change_validated_invoice_currency(self):
         inv = self.create_simple_invoice(fields.Date.today())
+        inv.track_created_invoice()
```

Comment:
If you are calling this method in the `create` why do you need to call it after the creation?

Reply:
I removed it. Done!